repo/meta/schema/config-schema.json

```diff
@@ -1,6 +1,4 @@
 {
-    "id": "http://json-schema.org/draft-04/schema#",
```

Comment:
This is not a reference to a schema, but a field in the JSON. Are there any other implications to changing this?

Reply:
Looks like someone messed up. This field is supposed to be a pointer to _your_ schema URL:

https://spacetelescope.github.io/understanding-json-schema/basics.html
```
Declaring a unique identifier
It is also best practice to include an id property as a unique identifier for each schema. For now, just set it to a URL at a domain you control, for example:

{ "id": "http://yourdomain.com/schemas/myschema.json" }
```